frontend/src/pages/modelServing/screens/projects/nimUtils.ts

```diff
@@ -77,3 +80,44 @@ export const getNIMServingRuntimeTemplate = async (
     return undefined;
   }
 };
+
+export const updatePVCName = (
```

Comment:
Can we get a unit test to make sure this performs the changes as you see them? Simple providing different sets of information and asserting the response. Also helps not keep reducing the test coverage we have with new NIM code.

Reply:
Unit test added.